### PR TITLE
Move StockController into Admin/Sell/Catalog/

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Moved StockController into the appropriate namespace
| Type?             | refacto
| Category?         | BO
| BC breaks?        | yes - The file changed its namespace and location
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | The stock page loads
| Possible impacts? | This only impacts the page load because the content is handled by another controller

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25561)

### CODE REVIEW

**Review by Progi1984 — COMMENTED** [review]
@eternoendless Thank you for your contribution. Could you fix PHPStan error ?

File: tests/Resources/modules/ps_banner/config.xml [hunk 1]
```diff
@@ -5,7 +5,8 @@
     <version><![CDATA[2.1.0]]></version>
     <description><![CDATA[Displays a banner on your shop.]]></description>
     <author><![CDATA[PrestaShop]]></author>
+    <tab><![CDATA[]]></tab>
     <is_configurable>1</is_configurable>
     <need_instance>0</need_instance>
 	<limited_countries></limited_countries>
-</module>
+</module>
```
Comment (matthieu-rolland) [inline tests/Resources/modules/ps_banner/config.xml:12]:
```suggestion
</module>

```

**Review by matthieu-rolland — COMMENTED** [review]
minor feedback and phpstan issue, otherwise lgtm 👍

**Review by matks — DISMISSED** [review]

**Review by sowbiba — COMMENTED** [review]
Approved as soon as phpstan error fixed. Maybe remove return annotation from FrameworkBundleAdminController::overviewAction ?

**Review by matks — CHANGES_REQUESTED** [review]
Need to fix PHPStan

**Comment by PierreRambaud** [comment]
Hi @eternoendless ,

Since we had no news from you for more than 30 days, I'll close this pull request. Feel free to reopen or open another one if you think it's still relevant.

Thanks!

**Comment by eternoendless** [comment]
Reopened here #26836

**Pull request closed without merging**